docs/manual.rst

```diff
@@ -1098,9 +1098,37 @@ differently.
   >>> b == c
   False
 
-.. method:: object.equals_exact(other, tolerance)
+.. method:: object.equals_exact(other, tolerance=0.0, normalize=False)
 
-  Returns ``True`` if the object is within a specified `tolerance`.
+    Returns ``True`` if the geometries are structurally equivalent within a
+    given tolerance.
+
+    This method uses exact coordinate equality, which requires coordinates
+    to be equal (within specified tolerance) and in the same order for
+    all components (vertices, rings, or parts) of a geometry. This is in
+    contrast with the :meth:`~object.equals` function which uses spatial
+    (topological) equality and does not require all components to be in the
+    same order. Because of this, it is possible for :meth:`~object.equals` to
+    be ``True`` while :meth:`~object.equals_exact` is ``False``.
+
+    The order of the coordinates can be normalized (by setting the `normalize`
```

Comment:
```suggestion
    The order of the coordinates can be normalized (by setting the ``normalize``
```

Reply:
If shapely follows numpy doc recommendations (which I think it is supposed to), then function arguments should be referenced with a single `

Reply:
@jorisvandenbossche - perhaps you can answer this one?

Reply:
Either way, I still wanted review from @jorisvandenbossche before this gets merged.  Sorry for the delay in proceeding with this.

Reply:
@brendan-ward maybe someone else could approve this ? It's a bit sad that this PR stalls for a such a minor issue ...